components/turret.py

```diff
@@ -71,34 +49,30 @@ def setup(self) -> None:
         self.motor.config_kI(0, self.pidI, 10)
         self.motor.config_kD(0, self.pidD, 10)
         self.motor.configAllowableClosedloopError(0, self.CLOSED_LOOP_ERROR, 10)
+        self.scan_increment_degrees = 10.0
```

Comment:
Is there a particular reason to store this in degrees and convert to radians each time we're using this?

Reply:
Fixed